src/common/AndroidLauncher.ts

```diff
@@ -89,6 +90,7 @@ export class AndroidLauncher {
                     CommonUtils.updateCliAction(messages.getMessage('launchAppStatus', [targetApp]));
 
                     const launchActivity = appConfig?.activity ?? '';
+                    const target = launchActivity ? `${targetApp}/${launchActivity}` : targetApp;
```

Comment:
Launching an app on a device is now simpler. It is done using a Device object, which under the hood will use iOS/Android utils. I made the code such that only a target is passed in. So on iOS the caller would need to pass in bundle id and on android the caller would pass in package name + target activity (rather than having a different method signature on Android to pass package name & target activity separately)